src/main/java/frc/robot/Constants.java

```diff
@@ -137,12 +137,31 @@ public AimModes previous() {
 
   // AUTONOMOUS Constants
 
-  // Path json files
-  public static final String taxiPath = "paths/TaxiOut.wpilib.json";
-  public static final String oneBallPath = "paths/TaxiOutFromFender.wpilib.json";
-  public static final String twoBallPath[] = {
-    "paths/TaxiOutToGrabBall.wpilib.json", "paths/GoBackIntoFender.wpilib.json"
-  };
+  public enum Auton {
+    TAXI("paths/TaxiOut.wpilib.json"),
+    ONEBALL("paths/TaxiOutFromFender.wpilib.json"),
+    TWOBALL("paths/TaxiOutToGrabBall.wpilib.json", "paths/GoBackIntoFender.wpilib.json"),
+    THREEBALL,
+    FOURBALL,
+    FIVEBALL,
+    TEST(
+        "paths/FlippedVectorTest1.wpilib.json",
+        "paths/FlippedVectorTest2.wpilib.json"); // change according to what path you want to test
+
+    private String paths[];
+
+    private Auton() {
+      paths = null;
+    }
+
+    private Auton(String... paths) {
+      this.paths = paths;
+    }
```

Comment:
Just an FYI, since paths is already optional, you don't need to make the Auton() method with no parameters. Fine as is though.